client/components/timezone-dropdown/docs/example.jsx

```diff
@@ -34,7 +34,7 @@ export default React.createClass( {
 					<a href="/devdocs/design/timezone-dropdown">TimezoneDropdown</a>
 				</h2>
 
-				<Card style={ { width: '300px', margin: 0 } }>
+				<Card style={ { width: '300px', height: '350px', margin: 0 } }>
```

Comment:
I am not sure this is the solution here.
Probably dropdown needs some Z-index magic, because if we use it in settings, we wont be making vertical space for it, we want it to go over the other settings.